lib/roda/plugins/exception_page.rb

```diff
@@ -163,6 +184,14 @@ def self.js
 END
       end
 
+      module ClassMethods
+        # Read or set the CSS to render in the exception page's CSS file
+        attr_accessor :exception_page_css
```

Comment:
Having these be accessors is a trade-off; we're now retaining the memory at all times for the exception page, even in development mode. However, we're not generating the string anew for each exception.

To do this, we're going to need to retain _something_. In trying to do this carefully and in the way that I've understood you want Roda to be written, I'm becoming more aware that overriding the method has the smallest footprint on memory.

I wonder if defining the method programmatically would be better (e.g. use the current methods to define defaults on `ClassMethods` which can then be overridden on a per-app basis)? That certainly becomes more cumbersome and harder to read and understand, but it would be more efficient for the computer.

Reply:
I think it's probably best to define these as instance methods, not class methods.  That allows for pre-request behavior (the most flexible).  By default, the methods just call `ExceptionPage.#{css,js}`